vocab/types.json

```diff
@@ -1376,15 +1376,15 @@
         ]
     },
     "https://openminds.ebrains.eu/core/RORID": {
-        "description": null,
+        "description": "A persistent identifier for a research organization, provided by the Research Organization Registry.",
         "label": "RORID",
         "schemas": [
             "core/v3/miscellaneous/RORID",
             "core/v4/digitalIdentifier/RORID"
         ]
     },
     "https://openminds.ebrains.eu/core/RRID": {
-        "description": null,
+        "description": "A persistent identifier for a research resource provided by the Resource Identification Initiative.",
```

Comment:
I suggest to follow the same logic as in "Structured information about a digital object identifier, as standardized by the International Organization for Standardization."